src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -114,26 +121,31 @@ def validate_ip_ranges(namespace):
 
     restrict_traffic_to_agentnodes = "0.0.0.0/32"
     allow_all_traffic = ""
-    ip_ranges = [ip.strip() for ip in namespace.api_server_authorized_ip_ranges.split(",")]
+    ip_ranges = [ip.strip()
```

Comment:
Why did these get updated? Auto saving?

Reply:
Yea, I think the lines were too long, so the autoformatter split them up.

Reply:
Hmm, can you check the documentation. Or try and see how many chars are allowed. I don't love splitting these up so I want to be sure it is required

Reply:
Yea sure. Do you know a fast way to revert them?